packit_service/models.py

```diff
@@ -537,6 +537,35 @@ def set_build_logs_url(self, build_logs: str):
     def get_project(self) -> GitProjectModel:
         return self.job_trigger.get_trigger_object().project
 
+    @property
+    def api_structure(self) -> Dict[str, Any]:
```

Comment:
This seems to me to be a serialisation exercise: are there other ways to do this, without having to write (and then maintain) ~30 lines of code?

Reply:
- I wanted to have it in one place (unlike the koji build implementation).
- I wanted to control what's there.
  - Not all attributes make sense.
  - Handling with triggers needs to be done manually for now. (We can use `trigger` and nested serialization instead.) 
- It makes sense to do this change for all at once and have some design when providing payload with nested objects.